src/cmd-buildextend-metal

```diff
@@ -67,19 +72,20 @@ if [ $# -ne 0 ]; then
 fi
 
 # default to both BIOS and UEFI for non-s390x and BIOS DASD for s390x
-if [ -z "${BIOS}" ] && [ -z "${UEFI}" ] && [ -z "${DASD}" ]; then
-    BIOS=1
+if [ -z "${BIOS}" ] && [ -z "${UEFI}" ] && [ -z "${DASD}" ] && [ -z "${ZFCP}" ]; then
     if [ "${arch}" != "s390x" ]; then
+        BIOS=1
         UEFI=1
     else
         DASD=1
+        ZFCP=1
     fi
 fi
 
-if [ -n "${UEFI}" ] && [ "${arch}" = "s390x" ]; then
-    fatal "${arch} is not supported for building UEFI image"
-elif [ -n "${DASD}" ] && [ "${arch}" != "s390x" ]; then
-    fatal "${arch} is not supported for building DASD image"
+if ([ -n "${BIOS}" ] || [ -n "${UEFI}" ]) && [ "${arch}" = "s390x" ]; then
```

Comment:
Shellcheck is complaining about this:

```
In src/cmd-buildextend-metal line 85:

if ([ -n "${BIOS}" ] || [ -n "${UEFI}" ]) && [ "${arch}" = "s390x" ]; then

   ^-- SC2235: Use { ..; } instead of (..) to avoid subshell overhead.

In src/cmd-buildextend-metal line 87:

elif ([ -n "${DASD}" ] || [ -n "${ZFCP}" ]) && [ "${arch}" != "s390x" ]; then

     ^-- SC2235: Use { ..; } instead of (..) to avoid subshell overhead.

```

Reply:
Ouch.